core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2468,4 +2468,15 @@ package object config {
       .version("3.4.0")
       .booleanConf
       .createWithDefault(false)
+
+  private[spark] val RDD_CACHE_VISIBILITY_TRACKING_ENABLED =
+    ConfigBuilder("spark.rdd.cache.visibilityTracking.enabled")
+      .internal()
+      .doc("Set to be true to enabled RDD cache block's visibility status. Once it's enabled," +
+        " a RDD cache block can be used only when it's marked as visible. And a RDD block will be" +
+        " marked as visible only when one of the tasks generating the cache block finished" +
+        " successfully. This is relevant in context of consistent accumulator status.")
+      .version("3.4.0")
```

Comment:
According to the commit history, it seems this pr lands at master branch(3.5.0). Should this backport to 3.4.0 or update the version ?

Reply:
No, we should mark this as 3.5.0 instead ... slipped through the review (it was created way before 3.4 was cut).
Can you create a follow up to change this to 3.5 @ivoson ? Thx

Reply:
Sure, will do.